dev-tools/mage/pytest.go

```diff
@@ -144,7 +144,7 @@ func PythonTest(params PythonTestArgs) error {
 	}
 
 	pytestOptions := []string{
-		"--timeout=90",
+		"--timeout=120",
```

Comment:
please let's not change non buildkite related things related to flakiness in this PR. A future PR should be raised (and the corresponding team would need to review it).

Reply:
Yes, sure. I'll remove it,